### DISCUSSION TITLE
how to reset settings?

### DISCUSSION THREAD

**Discussion by av100** [darkreader/darkreader#6974] [Q&A]
firefox 92

https://www.reddit.com/r/firefox/comments/hqggml/remove_addon_settings/  did not help

**Comment by Gusted** [comment]
Hi!

Please first enable to new UI: 

    Click Dark Reader icon.
    Click Dev tools (in the bottom-right corner).
    Click Preview new design.

Then you can go to:
    Click Dark Reader icon.
    Click Settings.
    Click Manage settings.
    Click Reset Settings.

**Reply by av100** [reply]
> Hi!
> 
> Please first enable to new UI:
> 
> ```
> Click Dark Reader icon.
> Click Dev tools (in the bottom-right corner).
> Click Preview new design.
> ```
> 
> Then you can go to: Click Dark Reader icon. Click Settings. Click Manage settings. Click Reset Settings.

The new UI does not apply. The extension broke and I can not reset its settings.
It is impossible to save extension settings

**Reply by Gusted** [reply]
That seems like you've been hit by a bug which is fixed in the next release.

**Comment by av100** [comment]
https://imgur.com/a/mFRfuQg
